templates/shared_services/firewall/terraform/firewall.tf

```diff
@@ -13,6 +13,8 @@ resource "azurerm_firewall" "fw" {
   name                = "fw-${var.tre_id}"
   resource_group_name = local.core_resource_group_name
   location            = data.azurerm_resource_group.rg.location
+  sku_tier            = "Standard"
+  sku_name            = "AZFW_VNet"
```

Comment:
Could this cause the firewall to be recreated? If so, is that something that we should call out anywhere?

Reply:
Cool - might be good to add a custom rule before deploying latest to check it is preserved?

Reply:
Tested that and firewall didn't get recreated 👍